wmfdata/spark.py

```diff
@@ -32,5 +69,9 @@ def get_spark_session(master='yarn', app_name='wmfdata', spark_config={}):
     for k, v in spark_config.items():
         builder.config(k, v)
 
-    spark = builder.getOrCreate()
-    return spark
+    session = builder.getOrCreate()
```

Comment:
This is fine, but for consistency the SparkSession instance is almost always called 'spark'.  It is called this in both the pyspark and scala CLI repls, and is instantiate as such in the Jupyter spark kernels too.

Reply:
Hmm, yeah. I could change this.

Perhaps the bigger issue is that the module itself is called `spark`. I usually start my notebooks with `from wmfdata import hive` or similar, so following that pattern and doing `from wmfdata import spark` would conflict with that convention. Would it be better to call this module `spark_tools`? Or perhaps we could just recommend starting notebooks with `import wmfdata as wmf` and use `wmf.spark`?

Reply:
Ah right.  I dunno, maybe just call this spark_session to be clear?

Reply:
On reflection, I prefer to leave it as it is. I use `session` consistently throughout this module; adding Spark seems unnecessary because this is the `spark` module (that is also why I renamed `get_spark_session` to `get_session`. When using this module, it will be perfectly possible to do:
```
import wmfdata as wmf

spark = wmf.spark.get_session()
```

That seems clear, concise, and consistent with the wider practice to me. I'll actually switch my own practice to using `wmf.spark`, `wmf.hive`, and so on to avoid confusion in the future.